src/modules/user/commands/delete-user/delete-user.service.ts

```diff
@@ -1,13 +1,12 @@
 import { UserRepositoryPort } from '@modules/user/database/user.repository.port';
 import { Inject } from '@nestjs/common';
 import { CommandHandler } from '@nestjs/cqrs';
-import { UserRepository } from '../../database/user.repository';
 import { DeleteUserCommand } from './delete-user.command';
 
 @CommandHandler(DeleteUserCommand)
 export class DeleteUserService {
   constructor(
-    @Inject(UserRepository)
+    @Inject("userRepositoryPort")
```

Comment:
inject the port not the implementation, because the impl contains infrastructure related code